### PR TITLE
Add vesper

### PR DESCRIPTION
**Pull request by FormalSnake**
Add the Vesper theme.
https://github.com/raunofreiberg/vesper

I tried adding it myself but I do not think it is 100% accurate, especially since I don't know how to test them.
If someone experienced can finish it that would be appreciated.
(I am using tmux and nvim)

### CODE REVIEW

File: base16/vesper.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+name: "Vesper"
+author: "FormalSnake (https://github.com/formalsnake)"
+variant: "dark"
+palette:
+  base00: "101010"
+  base01: "232323"
+  base02: "222222"
+  base03: "333333"
+  base04: "999999"
+  base05: "c1c1c1"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base05: "b7b7b7"
```

File: base16/vesper.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  base06: "999999"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base06: "c1c1c1"
```

File: base16/vesper.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  base07: "c1c1c1"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base07: "d5d5d5"
```

File: base16/vesper.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  base08: "5f8787"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base08: "de6e6e"
```

File: base16/vesper.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  base09: "aaaaaa"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base09: "dab083"
```

File: base16/vesper.yaml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  base0A: "FFC799"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base0A: "ffc799"
```

File: base16/vesper.yaml [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  base0B: "fbcb97"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base0B: "5f8787"
```

File: base16/vesper.yaml [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  base0C: "aaaaaa"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base0C: "60a592"
```

File: base16/vesper.yaml [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  base0D: "888888"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base0D: "8eaaaa"
```

File: base16/vesper.yaml [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+  base0E: "999999"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base0E: "d69094"
```

File: base16/vesper.yaml [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+  base0F: "444444"
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
  base0F: "876c4f"
```

File: base16/vesper.yaml [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (JamyGolden) [inline base16/vesper.yaml]:
```suggestion
```

**Review by JamyGolden — COMMENTED** [review]
I've changed it slightly based on the colours you have in your vscode template. I've also changed some of the tinting of the colours to more closely match the colours generally used: image

I've added some suggested line changes, but it changes the file to:

```yaml
system: "base16"
name: "Vesper"
author: "FormalSnake (https://github.com/formalsnake)"
variant: "dark"
palette:
  base00: "101010"
  base01: "232323"
  base02: "222222"
  base03: "333333"
  base04: "999999"
  base05: "b7b7b7"
  base06: "c1c1c1"
  base07: "d5d5d5"
  base08: "de6e6e"
  base09: "dab083"
  base0A: "ffc799"
  base0B: "5f8787"
  base0C: "60a592"
  base0D: "8eaaaa"
  base0E: "d69094"
  base0F: "876c4f"
```

With the above changes, an example of a shell script using base16-vim: 

image

**Comment by FormalSnake** [comment]
Nice! Thank you very much!

**Pull request merged**